README.md

```diff
@@ -1,150 +1,119 @@
-        ~~~
-
-        and look at http://localhost:4000.
+4.  Select the owner for your new repository.
+    (This will probably be you, but may instead be an organization you belong to.)
 
-8.  Commit your changes and push to the `gh-pages` branch of your
-    repository.
+5.  Choose a name for your workshop website repository.
+    This name should have the form `YYYY-MM-DD-site`,
+    e.g., `2015-07-01-miskatonic`.
 
-9.  Send the workshop coordinators the URL for your GitHub repository.
+6.  Make sure the repository is public.
 
-If the identifier for your workshop is `2015-07-01-esu`, and your
-GitHub username is `ghopper`, your workshop repository will be
-`https://github.com/ghopper/2015-07-01-esu` and the website for your
-workshop will be `https://ghopper.github.io/2015-07-01-esu`.  (Note:
-the workshop coordinators will want the former URL, not the latter.)
+7.  At this point, you should have a page like this:
 
-## Manual Setup
+    ![](http://software-carpentry.org/img/workshop-template/using-github-import.png)
 
-You can set up your repository manually instead of using the automated
-`create` script.  As above, we will assume that your user ID is
-`ghopper` and the identifier for your workshop is `2015-07-01-esu`.
+    You can now click "Begin Import".
+    When the process is done,
+    you can click "Continue to repository" to visit your newly-created repository.
 
-1.  Create an empty repository on GitHub called `2015-07-01-esu`.
+## Customizing Your Website
 
-2.  Clone the template repository to your computer in a directory with
-    the same name as your workshop identifier:
-
-    ~~~
-    $ git clone -b gh-pages -o upstream https://github.com/swcarpentry/workshop-template.git 2015-07-01-esu
-    ~~~
-
-3.  Go into that directory using
-
-    ~~~
-    $ cd 2015-07-01-esu
-    ~~~
+1.  Go into your newly-created repository,
+    which will be at `http://github.com/your_username/YYYY-MM-DD-site`.
+    For example,
+    if `your_username` is `gvwilson`,
+    the repository's URL will be `https://github.com/gvwilson/2015-07-01-mistaktonic`.
 
-4.  Add your GitHub repository as a remote called `origin` using
+2.  Edit `index.html`.
+    You can do this in the browser by clicking on it in the file view
+    and then selecting the pencil icon in the menu bar:
 
-    ~~~
-    $ git remote add origin https://github.com/ghopper/2015-07-01-esu.git
-    ~~~
+    ![](http://software-carpentry.org/img/workshop-template/edit-index-file-menu-bar.png)
 
-5.  Edit `index.html`.  Hints are embedded in the file, and full
-    instructions are in [CUSTOMIZATION.md](CUSTOMIZATION.md).
+    or you can clone the repository to your desktop,
+    edit `index.html` there,
+    and push your changes back to the repository.
 
-6.  If you have installed the software described below:
+3.  When you are done editing,
+    you can preview your website at its GitHub Pages URL,
+    which is `http://your_id.github.com/YYYY-MM-DD-site`.
```

Comment:
On Wed, Jan 21, 2015 at 04:46:42AM -0800, Carol Willing wrote:
> > -6.  If you have installed the software described below:
> > +3.  When you are done editing,
> > +    you can preview your website at its GitHub Pages URL,
> > +    which is `http://your_id.github.com/YYYY-MM-DD-site`.
> 
> Perhaps nit-picky since your_id.github.com redirects to
> your_id.github.io but there is an inconsistency between line 75 and
> 78.

We should use the github.io form [1].

[1]: https://github.com/swcarpentry/workshop-template/pull/138/#issuecomment-70687920